docs-new/home/ASF.md

```diff
@@ -0,0 +1,18 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+Apache Software Foundation links
```

Comment:
Is this missing content? Or was it just not accessible in the old site?

Reply:
I need to play with this again, mkdocs wasn't liking that I had an entire section without any markdown file and only external links so I added this as an index, I imagine there's some trick to this to not do what I'm doing now.